caffe2/CMakeLists.txt

```diff
@@ -278,9 +278,8 @@ if(USE_ROCM)
   target_include_directories(caffe2_hip PRIVATE ${Caffe2_HIP_INCLUDES})
   target_include_directories(caffe2_hip INTERFACE $<INSTALL_INTERFACE:include>)
 
-  IF(BUILD_ATEN)
-    aten_set_target_props(caffe2_hip)
-  ENDIF()
+  # Set standard properties on the target
+  aten_set_target_props(caffe2_hip)
```

Comment:
Why was this conditional removed? It still seems applicable.

Reply:
@Jorghi12 for comment

Reply:
@ezyang Well if that's the case, then shouldn't this be a bug? https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L254

Reply:
I think your link bitrotted since you posted it, I don't know which line you're referring to now. (FYI, you can press 'y' on a page like this and it will link to a specific commit)

Reply:
@Jorghi12 did you mean to refer to https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L265?